src/metax_api/utils/reference_data_service.py

```diff
@@ -59,12 +59,18 @@ def _fetch_reference_data(cls, settings):
         esclient = Elasticsearch(settings['HOSTS'], **connection_params)
         indicesclient = IndicesClient(esclient)
         reference_data = {}
-
         for index_name, index_info in indicesclient.get_mapping().items():
             reference_data[index_name] = {}
             for type_name in index_info['mappings'].keys():
+                reference_data[index_name][type_name] = []
                 all_rows = scan(esclient, query={'query': {'match_all': {}}}, index=index_name, doc_type=type_name)
-                reference_data[index_name][type_name] = [ row['_source']['uri'] for row in all_rows if row['_source']['uri'] ]
+                for row in all_rows:
+                    entry = {}
+                    if row['_source']['uri']:
+                        entry['uri'] = row['_source']['uri']
+                        if row['_source']['code']:
```

Comment:
This seems to crash while running local tests:
 File "/metax/metax-api/src/metax_api/utils/reference_data_service.py", line 71, in _fetch_reference_data
    if row['_source']['code']:
KeyError: 'code'
... which is odd, since Travis tests passed.

Reply:
Does your local env ES have the codes in the indices? If you have not reindexed for a while then sure, but on the other hand that code shouldn't fail in any case.

Reply:
I did reindex, but I forgot the new stuff comes from metax-ops and not in this repo. With new ref data there was no problem.